pages/onboarding_new_cluster.ipynb

```diff
@@ -0,0 +1,826 @@
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "#### Define cluster in ACM\n",
+    "\n",
+    "In order to use ACM with ArgoCD we have to create manifests which defines if cluster will use ArgoCD. This file is going to be located in `acm/overlays/$CLUSTER_REGION/infra/klusterletaddonconfigs`.\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "source": [
+    "text_input=\"\"\"\\\n",
+    "apiVersion: agent.open-cluster-management.io/v1\n",
+    "kind: KlusterletAddonConfig\n",
+    "metadata:\n",
+    "  name: %s\n",
+    "  namespace: %s\n",
+    "spec:\n",
+    "  applicationManager:\n",
+    "    argocdCluster: true\n",
+    "    enabled: true\n",
+    "\"\"\" % (CLUSTER_NAME, CLUSTER_NAME)\n",
+    "%store text_input >acm/overlays/$CLUSTER_REGION/infra/klusterletaddonconfigs/{CLUSTER_NAME}.yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'acm/overlays/moc/infra/klusterletaddonconfigs/my-cluster.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Next step is to list this manifest in `kustomization.yaml` file which is located in  `acm/overlays/$CLUSTER_REGION/infra/`.\n",
+    "```\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "  ...\n",
+    "  - klusterletaddonconfigs/$CLUSTER_NAME.yaml\n",
+    "  ...\n",
+    "```\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "#### Enable logging via SSO\n",
+    "\n",
+    "As the last thing we have to setup logining to cluster via SSO:\n",
+    "Create secret for your cluster in `keycloak/overlays/$CLUSTER_REGION/infra/clients/$CLUSTER_NAME.yaml` and encrypt the file with sops. You can find the key to import from [here](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/prod/moc#secret-management):\n",
+    ":::{note}\n",
+    "Don't forget to remove unecrypted file\n",
+    ":::\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "source": [
+    "text_input=\"\"\"\\\n",
+    "apiVersion: keycloak.org/v3alpha1\n",
+    "kind: KeycloakClient\n",
+    "metadata:\n",
+    "    name: %s\n",
+    "    labels:\n",
+    "        client: %s\n",
+    "spec:\n",
+    "    client:\n",
+    "        clientId: %s\n",
+    "        defaultClientScopes:\n",
+    "            - profile\n",
+    "        description: %s\n",
+    "        name: %s cluster\n",
+    "        protocol: openid-connect\n",
+    "        secret: %s\n",
+    "        standardFlowEnabled: true\n",
+    "        redirectUris:\n",
+    "            - https://oauth-openshift.apps.$CLUSTER_NAME.smaug.operate-first.cloud/oauth2callback/operate-first\n",
```

Comment:
This URL is not right - missing %s for cluster name and %s instead of `smaug` for cluster region (which is `na` in case of MOC...

we probably need another variable with an if statement. MOC belongs to `moc` region and maps to `na.operate-first.cloud` dns and Hetzner is `emea` region and maps to `emea.operate-first.cloud` dns.

Reply:
Resolved in latest commit.

Reply:
Resolved in latest commit.